airflow-core/tests/unit/models/test_dagbag.py

```diff
@@ -65,7 +66,8 @@ def db_clean_up():
     db.clear_db_dags()
     db.clear_db_runs()
     db.clear_db_serialized_dags()
-    db.clear_dag_specific_permissions()
+    if "FabAuthManager" in conf.get("core", "auth_manager"):
+        db.clear_dag_specific_permissions()
```

Comment:
Similar to one of my previous comments, if a function is specific to FAB, I would handle it in the function and not in the function that is calling this function. Example:

```
def clear_dag_specific_permissions():
    if "FabAuthManager" in conf.get("core", "auth_manager"):
        return
```